wazo_webhookd/services/mobile/plugin.py

```diff
@@ -267,11 +268,22 @@ def run(
         data = event.get('data')
         name = event.get('name')
 
+        logger.debug('received event type %s with payload: %s', name, data)
         if notification_type := MAP_NAME_TO_NOTIFICATION_TYPE.get(name):
+            logger.debug(
+                'notification_type %s identified from event %s', notification_type, name
+            )
             return getattr(push, notification_type)(data)
         return None
```

Comment:
Would an error log be appropriate if we can't match any event type?

Reply:
You mean instead of the debug log, or in addition?

Reply:
I think the debug log is fine, but while we're here, we might as well add logging for unexpected code paths :)